week5.1-ex1.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+
+def files_start_with(directory_path):
+    """ Return list of the files, in the directory at the end of the path, start with deep"""
+    files = os.listdir(directory_path)  # get all the files in the directory in the end of the path
+    files_list = [file for file in files if file.startswith("deep")]
```

Comment:
reduce this to a single line of code

Reply:
![Capture](https://user-images.githubusercontent.com/13467749/160238868-b7fb4988-3482-4187-ae9c-0909f245637c.JPG)
type hints and return types